mala/network/network.py

```diff
@@ -296,6 +294,24 @@ def forward(self, inputs):
             inputs = layer(inputs)
         return inputs
 
+    def _append_activation_function(self, activation_function):
+        """
+        Append an activation function to the network.
+
+        Parameters
+        ----------
+        activation_function : str
+            Activation function to be appended.
+        """
+        if activation_function is None:
+            pass
+        elif isinstance(activation_function, str):
+            self.layers.append(
+                self._activation_mappings[activation_function]()
+            )
+        elif isinstance(activation_function, nn.Module):
+            self.layers.append(activation_function)
```

Comment:
This branch seems to allow `activation_function` to be something like `torch.nn.modules.activation.ReLU()` in addition to a string? If so, then one should maybe catch the case that `activation_function = torch.nn.modules.activation.ReLU` (class instead of instance), which for which the `isinstance()` test is False. I'd probably throw an error in that case.

Reply:
See above, both instances and classes are now allowed.